<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PidDialog</class>
 <widget class="QDialog" name="PidDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>单级PID</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="sb_sing_mi">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>kd</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>kp</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="sb_sing_kp">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>ki</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="QPushButton" name="bt_spd_pid_param">
            <property name="text">
             <string>速度环默认参数</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QDoubleSpinBox" name="sb_sing_mo">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="sb_sing_ki">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>maxOut</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="sb_sing_kd">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>maxInt</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QPushButton" name="bt_singang_pid_param">
            <property name="text">
             <string>单级角度环默认参数</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>串级PID</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>内环（速度环）</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="6" column="1">
              <widget class="QDoubleSpinBox" name="sb_in_mi">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QDoubleSpinBox" name="sb_in_mo">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>kd</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="sb_in_ki">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>ki</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>maxOut</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>kp</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="sb_in_kp">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="sb_in_kd">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>maxInt</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>外环（角度环）</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="6" column="1">
              <widget class="QDoubleSpinBox" name="sb_out_mi">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QDoubleSpinBox" name="sb_out_mo">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>kd</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="sb_out_ki">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>ki</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>maxOut</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>kp</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="sb_out_kp">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="sb_out_kd">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>maxInt</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="bt_casang_pid_param">
            <property name="text">
             <string>串级角度环默认参数</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="bt_ok">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
